File: mycollab-web/src/main/java/com/esofthead/mycollab/module/user/accountsettings/view/events/ProfileEvent.java
package com.esofthead.mycollab.module.user.accountsettings.view.events;

import com.esofthead.mycollab.vaadin.events.ApplicationEvent;

@SuppressWarnings("serial")
public class ProfileEvent {
	
	public static class GotoProfileView extends ApplicationEvent {
		public GotoProfileView(Object source, Object data) {
			super(source, data);
		}
	}
	
	public static class GotoProfileEdit extends ApplicationEvent {
		public GotoProfileEdit(Object source, Object data) {
			super(source, data);
		}
	}

	public static class GotoUploadPhoto extends ApplicationEvent {

		public GotoUploadPhoto(Object source, Object data) {
			super(source, data);
		}
	}
}

File: mycollab-web/src/main/scala/com/esofthead/mycollab/module/user/accountsettings/view/events/ProfileEvent.scala
package com.esofthead.mycollab.module.user.accountsettings.view.events

import com.esofthead.mycollab.eventmanager.ApplicationEvent

/**
 * @author MyCollab Ltd
 * @since 5.0.5
 */
object ProfileEvent {
  class GotoProfileView(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoUploadPhoto(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}
}
